include/vkgcDefs.h

```diff
@@ -328,6 +328,15 @@ struct ExtendedRobustness {
                            ///  valid to access and acts as if the descriptor is bounded to nothing.
 };
 
+/// Represents the tiling modes for compute shader thread group swizzling
```

Comment:
Please bump the version to 45.5 and add a comment in the beginning of this file (mention that you add a new enum type for CS swizzling).

LLPC_INTERFACE_MINOR_VERSION 4 -> LLPC_INTERFACE_MINOR_VERSION 5